rider/src/main/kotlin/com/jetbrains/rider/plugins/godot/run/GodotRunConfigurationGenerator.kt

```diff
@@ -70,21 +75,21 @@ class GodotRunConfigurationGenerator : LifetimedService() {
 
                     GodotProjectDiscoverer.getInstance(project).godot3Path.adviseNotNull(lt) { path ->
                         if (descriptor.isPureGdScriptProject){
-                            createOrUpdateGdScriptRunConfiguration(PLAYER_CONFIGURATION_NAME, "--path \"${basePath}\"", runManager, path, project)
-                            createOrUpdateGdScriptRunConfiguration(EDITOR_CONFIGURATION_NAME, "--path \"${basePath}\" --editor", runManager, path, project)
+                            createOrUpdateGdScriptRunConfiguration(PLAYER_CONFIGURATION_NAME, "--path \"${relPath}\"", runManager, path, project)
+                            createOrUpdateGdScriptRunConfiguration(EDITOR_CONFIGURATION_NAME, "--path \"${relPath}\" --editor", runManager, path, project)
                             return@adviseNotNull
                         }
-                        createOrUpdateRunConfiguration(PLAYER_CONFIGURATION_NAME, "--path \"${basePath}\"", runManager, path, project)
-                        createOrUpdateRunConfiguration(EDITOR_CONFIGURATION_NAME, "--path \"${basePath}\" --editor", runManager, path, project)
+                        createOrUpdateRunConfiguration(PLAYER_CONFIGURATION_NAME, "--path \"${relPath}\"", runManager, path, project)
+                        createOrUpdateRunConfiguration(EDITOR_CONFIGURATION_NAME, "--path \"${relPath}\" --editor", runManager, path, project)
                     }
                     GodotProjectDiscoverer.getInstance(project).godot4Path.adviseNotNull(lt) { path ->
                         if (descriptor.isPureGdScriptProject){
-                            createOrUpdateGdScriptRunConfiguration(PLAYER_CONFIGURATION_NAME, "--path \"${basePath}\"", runManager, path, project)
-                            createOrUpdateGdScriptRunConfiguration(EDITOR_CONFIGURATION_NAME, "--path \"${basePath}\" --editor", runManager, path, project)
+                            createOrUpdateGdScriptRunConfiguration(PLAYER_CONFIGURATION_NAME, "--path \"${relPath}\"", runManager, path, project)
+                            createOrUpdateGdScriptRunConfiguration(EDITOR_CONFIGURATION_NAME, "--path \"${relPath}\" --editor", runManager, path, project)
                             return@adviseNotNull
                         }
-                        createOrUpdateCoreRunConfiguration(PLAYER_CONFIGURATION_NAME, "--path \"${basePath}\"", runManager, path, project)
-                        createOrUpdateCoreRunConfiguration(EDITOR_CONFIGURATION_NAME, "--path \"${basePath}\" --editor", runManager, path, project)
+                        createOrUpdateCoreRunConfiguration(PLAYER_CONFIGURATION_NAME, "--path \"${relPath}\"", runManager, path, project)
+                        createOrUpdateCoreRunConfiguration(EDITOR_CONFIGURATION_NAME, "--path \"${relPath}\" --editor", runManager, path, project)
```

Comment:
_(Note)_ Pro tip™: to avoid awkward quoting, take a look at `ParameterListUtil` class in IntelliJ. My recommendation is to operate argument lists rather than paths.

(I see that it was here before this PR, so not a concern, just a note.)